integration/geo_test.go

```diff
@@ -757,9 +907,23 @@ func TestGeo(t *testing.T) {
 		succRound3("GEORADIUS", "stations", -73.9718893, 40.7728773, 4, "km", "ASC", "COUNT", 3),
 		succRound3("GEORADIUS", "stations", -73.9718893, 40.7728773, 4, "km", "ASC", "COUNT", 99999),
 
+		succRound3("GEORADIUSBYMEMBER", "stations", "Astor Pl", 20, "km"),              // This testcase doesn't pass with 4 km. I had expected it to pass with 8km at least. 20km seems to do it. How can 4k below pass, when 4k here doesn't?
+		succRound3("GEORADIUSBYMEMBER", "stations", "Astor Pl", 20, "km", "WITHDIST"),  // This testcase doesn't pass with 4 km.
+		succRound3("GEORADIUSBYMEMBER", "stations", "Astor Pl", 20, "km", "WITHCOORD"), // This testcase doesn't pass with 4 km.
```

Comment:
These 3 testcases needed 20 km radius instead of 4 km. Not sure why.

Reply:
The order is undefined:
>The command default is to return unsorted items. Two different sorting methods can be invoked using the following two options:

I added some commands to sort results. Now they work fine with 4km. I guess that with 20Km the lookup order works out differently.